docs/api/en/animation/AnimationMixer.html

```diff
@@ -73,7 +73,7 @@ <h3>[method:Object3D getRoot]()</h3>
 		<p>Returns this mixer's root object.</p>
 
 		<h3>[method:this stopAllAction]()</h3>
-		<p>Deactivates all previously scheduled actions on this mixer.</p>
+		<p>Deactivates all previously scheduled actions on this mixer, calling [page:AnimationAction.stop]() on all actions.</p>
```

Comment:
Why is it important for the user to know that `stopAllAction()` internally uses `stop()`?

TBH, I in general prefer to not leak such implementation details in the API documentation.

Reply:
The reason is because the uncache method mentions that `.stop()` should be called on all actions, and `stopAllAction()` does that. It wasn't obvious, and better if the docs tell what needs to be known than having to open the source code.

Reply:
Then we can update the uncache method so it says that, no?

Reply:
I've updated the PR to avoid leaking implementation details of `stopAllAction()`. `uncacheRoot()` should be documented now more clear than before.